taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ["license_number"]
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if (len(license_number) == 8 and
+                license_number[:3].isalpha() and
+                license_number[:3].isupper() and
+                license_number[3:].isdigit()):
+            return license_number
+        raise ValidationError("Invalid licence number.")
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("first_name",
+                                                 "last_name",
+                                                 "license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if (len(license_number) == 8 and
+                license_number[:3].isalpha() and
+                license_number[:3].isupper() and
+                license_number[3:].isdigit()):
+            return license_number
+        raise ValidationError("Invalid licence number.")
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
Good job on using `get_user_model()` here. This is the correct way to get the user model.